variables.json

```diff
@@ -123,6 +123,43 @@
     "schematron_report": "validations/padigital_missing_thumbnailURL.sch",
     "xsl_branch": "master",
     "xsl_filename": "transforms/pennstateingest.xsl",
+    "xsl_repository": "tulibraries/aggregator_mdx"
+      },
+  "USCRANTON_OAI_CONFIG": {
+    "endpoint": "http://digitalservices.scranton.edu/oai/oai.php",
+    "md_prefix": "oai_qdc",
+    "all_sets": "False",
+    "excluded_sets": [],
+    "included_sets": [
+      "aquinas",
+      "basketball",
+      "commencement",
+      "costello",
+      "ics",
+      "medieval",
+      "p15111coll2",
+      "p15111coll4",
+      "p16214coll1",
+      "p9000coll4",
+      "p9000coll6",
+      "p9000coll7",
+      "passionists",
+      "prpubs",
+      "publications",
+      "scholarship",
+      "SCRYB03",
+      "zanerbloser",
```

Comment:
the ending comma should be deleted after zanerbloser along with the extra new line below